Vagrantfile

```diff
@@ -0,0 +1,32 @@
+# set up a Vagrantbox to run the repl
+# check it out at http://phpepl.dev after running 'vagrant up'
```

Comment:
I'd like to add some of these details to the readme so users can take advantage of this awesome setup. Is it as simple as having vagrant installed, cd'ing into the phpepl root and then running vagrant up?